3rdparty/python/requirements.txt

```diff
@@ -1,6 +1,7 @@
 ansicolors==1.1.8
 fasteners==0.16.3
 freezegun==1.1.0
+humanize==3.12.0
```

Comment:
Adding a third-party dependency is usually frowned upon because it increases the time to install Pants. That is particularly painful for CI, where the installation of Pants is often slow. It looks like `humanize` has prebuilt wheels, so it's not quite as egregious. But still something to consider.

Also, increase the surface area of Pants's supply chain for security.

I wonder if instead you could use the utility we already have? If not, develop a new one that ~vendors the parts of `humanize` you want?

https://github.com/pantsbuild/pants/blob/f55cb5aa68f3b55cbe5de1509569e86fa3a0b0d4/src/python/pants/util/strutil.py#L91-L100

Reply:
Yeah I'll look into doing that. Thanks!

Reply:
Also, https://github.com/pantsbuild/pants/pull/13498

Reply:
Lastly <insert joke about JS/TS gobbling up dependencies like candy>